src/ui/screens/settings/ViewNotifications.qml

```diff
@@ -20,22 +20,6 @@ MZViewBase {
     _viewContentData: ColumnLayout {
         spacing: MZTheme.theme.windowMargin * 1.5
 
-        MZCheckBoxRow {
-            id: captivePortalAlert
-            objectName: "settingCaptivePortalAlert"
-            visible: MZFeatureList.get("captivePortal").isSupported
-            //% "Guest Wi-Fi portal alert"
-            labelText: qsTrId("vpn.settings.guestWifiAlert")
-            //% "Get notified if a guest Wi-Fi portal is blocked due to VPN connection"
-            subLabelText: qsTrId("vpn.settings.guestWifiAlert.description")
-            isChecked: (MZSettings.captivePortalAlert)
-            showDivider: false
-            onClicked: {
-                MZSettings.captivePortalAlert = !MZSettings.captivePortalAlert
```

Comment:
Should the captivePortalAlert related Settings code be removed too?

Reply:
Another option is to keep this UI which controls if the ScreenCaptivePortal screen should be shown when the captive portal detection is run when the Connection manager state changes or when the network changes.